templates/etc/init.d/deb/td-agent

```diff
@@ -184,9 +184,6 @@ do_reload() {
 }
 
 do_restart() {
-  if ! do_configtest; then
```

Comment:
Careful — this disables other, legitimate use cases, like making sure the rest of the config works. Can we instead make sure the Prometheus config chunk doesn't break the check?

Reply:
See #230 for an alternative solution that doesn't have this disadvantage.

Reply:
That one looks better. Just approved.